apps/test/util/testUtils.js

```diff
@@ -278,11 +285,16 @@ export function throwOnConsoleWarnings() {
         console.log('Unexpected call to console.warn: ' + msg);
         console.log(e.stack);
       }
-      throw new Error(msg);
+      // Store error so we can throw in after. This will ensure we hit a failure
+      // even if message was originally thrown in async code
+      firstErrorMessage = firstErrorMessage || msg;
```

Comment:
This breaks our tests for these, since console.warn and console.error no longer throw themselves (instead we throw in the after). I haven't been able to come up with a good alternative way to test. Any thoughts?